numpy/core/src/umath/ufunc_type_resolution.c

```diff
@@ -2228,6 +2235,23 @@ PyUFunc_DivmodTypeResolver(PyUFuncObject *ufunc,
             out_dtypes[3] = out_dtypes[0];
             Py_INCREF(out_dtypes[3]);
         }
+        /* divmod(m8[<A>], int) => m8[<A>], m8[<A>] */
+        else if (PyTypeNum_ISINTEGER(type_num2)) {
+            out_dtypes[0] = ensure_dtype_nbo(PyArray_DESCR(operands[0]));
+            if (out_dtypes[0] == NULL) {
+                return -1;
+            }
+            out_dtypes[1] = PyArray_DescrFromType(NPY_LONGLONG);
+            if (out_dtypes[1] == NULL) {
```

Comment:
`codecov` seems to be showing useful patch diffs again at least--not sure if this conditional is too pathological to test, but shows up as uncovered

Reply:
Oh, is that for platforms that don't support that integer size or something? I guess NumPy probably only uses one or maybe two platforms for the cov reports in any case.

Reply:
Hmmm, I copy pasted this without thinking maybe, or just wrote it without thinking. This condition can never be hit, it would be an allocation error, but that is not even possible...

Reply:
@tylerjereddy does codecov only show them if it thinks there is a serious change? It seemed a bit random whether or not the patch diff shows up in the CI grid.

Reply:
Maybe we need another setting or two for it to always show the `patch` diff--let's ping @thomasrockhu from their team.

I believe if there are some failures in CI, this can affect the reporting status for codecov. When we turned it back on recently though, it did show a patch entry, and just said something like "no tracked files affected" because we only changed the codecov config file itself. Anyway, now I'm side-tracking your PR..

Reply:
Hi everyone, taking a look at this now. Just to make sure I'm getting this right, sometimes the `patch` status check isn't showing up on PRs? If there are failures in CI, `Codecov` normally doesn't report back, but that setting can be overwritten. This would prevent `project` from showing up as well, though.

Reply:
Yeah, I think project shows up now always (or almost always?), patch seemed to be missing sometimes, not sure if due to failures. Do failures in the azure pipeline count, since we have some there fairly regularly unfortunately.

Reply:
So looks like for #16418, `patch` had still been set to `false`, and that's why no patch status showed up.

However, for more recent PRs that have a successful CI, I'm seeing a `patch` status. We, unfortunately, can't distinguish between what is and what isn't a CI status, so failures in Azure Pipelines do count. If you would prefer not to have this behavior, you can edit the `yaml` with

```
codecov:
  require_ci_to_pass: false
```